noise-maker.js

```diff
@@ -0,0 +1,44 @@
+const Synth= new Tone.Synth();
+Synth.toMaster();
+//Synth.triggerAttackRelease("C4","8n");
+
+const Green = document.getElementById("green");
+green.addEventListener("mousedown",e=>{
+  const note = e.target.dataset.note
+  Synth.triggerAttackRelease(note,"8n")
+});
+const Blue = document.getElementById("blue");
+blue.addEventListener("mousedown",e=>{
+  const note = e.target.dataset.note
+  Synth.triggerAttackRelease(note,"8n")
+});
+const Red = document.getElementById("red");
+red.addEventListener("mousedown",e=>{
+  const note = e.target.dataset.note
+  Synth.triggerAttackRelease(note,"8n")
+});
+const Yellow = document.getElementById("yellow");
+yellow.addEventListener("mousedown",e=>{
+  const note = e.target.dataset.note
+  Synth.triggerAttackRelease(note,"8n")
+});
```

Comment:
There seems to be some opportunity for condensing the code here. Similar to how you added an event listener for the keys. Since you have the e.target.dataset.note value, I think you could use querySelectorAll and add one selector for all the colored divs.

Reply:
You have to think how to simplify your code by using some loops